docs/source/notebooks_and_ipython/notebook-example/add_kedro_to_a_notebook.md

```diff
@@ -0,0 +1,668 @@
+)
+shuttles["d_check_complete"] = shuttles["d_check_complete"] == "t"
+shuttles["moon_clearance_complete"] = shuttles["moon_clearance_complete"] == "t"
+shuttles["price"] = (
+    shuttles["price"].str.replace("$", "").str.replace(",", "").astype(float)
+)
+rated_shuttles = shuttles.merge(reviews, left_on="id", right_on="shuttle_id")
+model_input_table = rated_shuttles.merge(companies, left_on="company_id", right_on="id")
+model_input_table = model_input_table.dropna()
+model_input_table.head()
+```
+
+```python
+# Model training
+from sklearn.model_selection import train_test_split
+
+X = model_input_table[
+    [
+        "engines",
+        "passenger_capacity",
+        "crew",
+        "d_check_complete",
+        "moon_clearance_complete",
+        "iata_approved",
+        "company_rating",
+        "review_scores_rating",
+    ]
+]
+y = model_input_table["price"]
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=3)
+
+from sklearn.linear_model import LinearRegression
+
+model = LinearRegression()
+model.fit(X_train, y_train)
+model.predict(X_test)
+```
+
+```python
+# Model evaluation
+from sklearn.metrics import r2_score
+
+y_pred = model.predict(X_test)
+r2_score(y_test, y_pred)
+```
+
+## Use Kedro for data processing
+Even if you’re not ready to work with a full Kedro project, you can still use its for data handling within an existing notebook project. This section shows you how.
+
+Kedro’s Data Catalog is a registry of all data sources available for use by the project. It offers a separate place to declare details of the datasets your projects use. Kedro provides [built-in datasets for different file types and file systems](/kedro_datasets) so you don’t have to write any of the logic for reading or writing data.
+
+Kedro offers a range of datasets, including CSV, Excel, Parquet, Feather, HDF5, JSON, Pickle, SQL Tables, SQL Queries, Spark DataFrames, and more. They are supported with the APIs of pandas, spark, networkx, matplotlib, yaml, and beyond. It relies on [`fsspec`](https://filesystem-spec.readthedocs.io/en/latest/) to read and save data from a variety of data stores including local file systems, network file systems, cloud object stores, and Hadoop. You can pass arguments in to load and save operations, and use versioning and credentials for data access.
+
+To start using the Data Catalog, you'll need a `catalog.yml` to define datasets that can be used when writing your functions. There is one included in the same folder as your notebook:
+
+
+
+
+```yaml
+companies:
+  type: pandas.CSVDataSet
+  filepath: data/companies.csv
+
+reviews:
+  type: pandas.CSVDataSet
+  filepath: data/reviews.csv
+
+shuttles:
+  type: pandas.ExcelDataSet
+  filepath: data/shuttles.xlsx
+```
+
+By using Kedro to load the `catalog.yml` file, you can reference the Data Catalog in your notebook as you load the data for data processing.
+
+```python
+# Using Kedro's DataCatalog
+
+from kedro.io import DataCatalog
```

Comment:
Are we assuming that the user has already installed `kedro` at this point?

Reply:
Good point! Added, but not here, I've put it earlier in the example.